bazel/foreign_cc/BUILD

```diff
@@ -198,7 +203,10 @@ envoy_cmake(
     lib_source = "@com_github_google_libsxg//:all",
     out_static_libs = ["libsxg.a"],
     tags = ["skip_on_windows"],
-    deps = ["@boringssl//:ssl"],
+    deps = envoy_select_boringssl(
```

Comment:
I am not a bazel expert, so this may be incorrect advice. But when I fixed this for WASM, I was able to use the already-defined alias `"@envoy//bazel:boringssl"` as the dependency without needing `envoy_select_boringssl`. Would that work in this case?

cc @keith who understands bazel better than me.

Reply:
yea it should work

Reply:
yeah, `bazel//:boringssl` works perfectly. Thanks for catching it.